server/game/player.js

```diff
@@ -307,8 +309,10 @@ class Player extends Spectator {
 
     resetCardPile(pile) {
         pile.each(card => {
-            card.moveTo('draw deck');
-            this.drawDeck.push(card);
+            if(pile !== this.cardsInPlay || !this.cardsInPlayBeforeSetup.includes(card)) {
```

Comment:
Something about this gives me a stomach ache but I'm not sure what else should be done for this weird card. 👌